src/shared/components/case-editor/domain/event-completion-return-states.enum.model.ts

```diff
@@ -0,0 +1,5 @@
+export enum EventCompletionReturnStates {
```

Comment:
Loving the use of enums 😊  however can we follow the typescript naming conventions. https://www.typescriptlang.org/docs/handbook/enums.html